framework/doc/content/source/kernels/ADMaterialPropertyValue.md

```diff
@@ -0,0 +1,24 @@
+# ADMaterialPropertyValue
```

Comment:
There isn't really any point to make this AD for output purposes, but we might as well leave it in, in case someone has a different application in mind.